components/org.wso2.micro.integrator.initializer/src/main/java/org/wso2/micro/integrator/initializer/deployment/AppDeployerServiceComponent.java

```diff
@@ -65,6 +69,23 @@ protected void activate(ComponentContext ctxt) {
         CAppDeploymentManager cAppDeploymentManager = new CAppDeploymentManager(configCtx.getAxisConfiguration());
         initializeDeployers(artifactDeploymentManager, cAppDeploymentManager);
 
+        // Register eventSink deployer
```

Comment:
Shall we move this code block to a method

Reply:
Fixed in bd10bd3